instrumentation/grape/lib/opentelemetry/instrumentation/grape/instrumentation.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module Grape
+      # The Instrumentation class contains logic to detect and install the Grape instrumentation
+      class Instrumentation < OpenTelemetry::Instrumentation::Base
+        # Minimum Grape version needed for compatibility with this instrumentation
+        MINIMUM_VERSION = Gem::Version.new('1.2.0')
```

Comment:
This is the minimum version required so we don't have to support [this workaround](https://github.com/DataDog/dd-trace-rb/blob/a02ccfc38123c087451f204ac6e822a6240216d5/lib/datadog/tracing/contrib/grape/endpoint.rb#L201).